linera-service/src/cli_wrappers/helm.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::PathBuf;
+
+use super::util::get_github_root;
+use crate::util::CommandExt;
+use anyhow::Result;
+use pathdiff::diff_paths;
+use tokio::process::Command;
+
+pub struct Helm;
+
+impl Helm {
+    pub async fn install(release_name: &str, configs_dir: PathBuf) -> Result<()> {
+        let github_root = get_github_root().await?;
+        let execution_dir = format!("{}/kubernetes/linera-validator", &github_root);
```

Comment:
why is it not a temporary directory?

Reply:
Because Helm needs to be executed from where the Chart file is at, otherwise it doesn't work